message/infrastructure/nats/streaming.go

```diff
@@ -0,0 +1,35 @@
+package nats
```

Comment:
I would recommend to rename file to `connection.go`, because it would be easier to find what can be found here